src/torchmetrics/wrappers/minmax.py

```diff
@@ -95,6 +96,10 @@ def compute(self) -> Dict[str, Tensor]:
         self.min_val = val if self.min_val.to(val.device) > val else self.min_val.to(val.device)
         return {"raw": val, "max": self.max_val, "min": self.min_val}
 
+    def forward(self, *args: Any, **kwargs: Any) -> Any:
+        """Use the original forward method of the base metric class."""
+        return super(WrapperMetric, self).forward(*args, **kwargs)
```

Comment:
this is redundant?
```suggestion
```

Reply:
No, it is needed for the tests to pass. The `Minmax` forward needs to call the `forward` of the `Metric` class. Without this it will call the forward of the `WrapperMetric` class which raises a `NotImplementedError`.